### PR TITLE
Update the `open_cabinet_door` and  `open_lever_handle_door` and objectives

### PR DESCRIPTION
**Pull request by pac48**
This PR updates the `open_cabinet_door` and  `open_lever_handle_door` and objectives to be compatible with the new behaviors ports in 5.0.

### CODE REVIEW

File: src/picknik_ur_site_config/objectives/open_lever_handle_door.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (EzraBrooks) [inline src/picknik_ur_site_config/objectives/open_lever_handle_door.xml:1]: unrelated change?

File: src/picknik_ur_site_config/objectives/open_lever_handle_door.xml [hunk 2]
```diff
@@ -24,13 +24,13 @@
...
+        <!--Whether task execution succeeds or fails, reset the controllers to disable admittance control.-->
         <Control ID="IfThenElse" name="execute_and_reset_controllers">
-          <!-- IF task execution succeeds -->
+          <!--IF task execution succeeds-->
           <Action ID="ExecuteMTCTask" solution="{open_lever_handle_door_solution}"/>
-          <!-- THEN reset the controllers -->
+          <!--THEN reset the controllers-->
           <Action ID="ActivateControllers" controller_names="/joint_trajectory_controller /robotiq_gripper_controller"/>
-          <!-- ELSE reset the controllers and return failure -->
+          <!--ELSE reset the controllers and return failure-->
```
Comment (EzraBrooks) [inline src/picknik_ur_site_config/objectives/open_lever_handle_door.xml:33]: unrelated changes?

Comment (pac48) [reply src/picknik_ur_site_config/objectives/open_lever_handle_door.xml]: Fixed

File: src/picknik_ur_site_config/objectives/open_lever_handle_door.xml (hunk 1 above)
Comment (pac48) [reply src/picknik_ur_site_config/objectives/open_lever_handle_door.xml]: Fixed

**Review by chancecardona — APPROVED** [review]
These are needed changes! Figure we can add more things if we find it as we go so best to get these in.

**Comment by chancecardona** [comment]
I think these should already be in no? Feel free to reopen if not.
